src/fts-backend-xapian.cpp

```diff
@@ -482,7 +482,7 @@ static int fts_backend_xapian_lookup(struct fts_backend *_backend, struct mailbo
 	/* Performance calc */
         gettimeofday(&tp, NULL);
         dt = tp.tv_sec * 1000 + tp.tv_usec / 1000 - dt;
-	i_info("FTS Xapian: %ld results in %ld ms",n,dt);	
+	i_debug("FTS Xapian: %ld results in %ld ms",n,dt);
```

Comment:
Not sure this is adequate, becuase if one need to put dovecot in debug mode, then all the debug messages comes, and fts debug gets lost. ALso performance data is quite usefull not in debug mode.